lib/tbd/psi.rb

```diff
@@ -1771,6 +1769,73 @@ def process(model = nil, argh = {})
       end
     end
 
+    # Unless a user has set the thermal bridge type of an individual edge via
+    # JSON input, reset any subsurface's head, sill or jamb edges as (mild)
+    # transitions if in close proximity other similar subsurface edges. User-set
+    # tolerance must be greater than default TBD tolerance (0.01 m). Both edges'
+    # origin and terminal vertices must be in close proximity. Edges of unhinged
+    # subsurfaces are ignored.
```

Comment:
A more detailed explanation (with visuals) will be introduced in the User Guide, most likely a new tab dedicated to subsurfaces.